File: binders/difido-python/difido/execution_details.py
'''
Created on Aug 10, 2017

@author: itai
'''

class ExecutionDetails(object):
    '''
    classdocs
    '''


    def __init__(self):
        '''
        Constructor
        '''
        self.description = ""
        self.execution_properties = {}
        self.shared = False
        self.force_new = False
        
    def dict(self):
        d = {}
        d['description'] = self.description
        d['executionProperties'] = self.execution_properties
        d['shared'] = self.shared
        d['forceNew'] = self.force_new   
        return d
        

File: binders/difido-nunit3/difido-common/Model/Execution/ExecutionDetails.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Difido.Model.Execution
{
    public class ExecutionDetails
    {
        public string description { get; set; }

        public Dictionary<string, string> executionProperties { get; set; }

        public bool shared { get; set; }

        public bool forceNew { get; set; }


        public ExecutionDetails()
        {
        }

        public ExecutionDetails(string description, bool shared = false, bool forceNew = false)
        {
            this.description = description;
            this.shared = shared;
            this.forceNew = forceNew;
        }

  



    }
}
